tests/Fixtures/app/AppKernel.php

```diff
@@ -110,6 +111,10 @@ protected function configureContainer(ContainerBuilder $c, LoaderInterface $load
 
         $loader->load(__DIR__."/config/config_{$this->getEnvironment()}.yml");
 
+        if (\PHP_VERSION_ID >= 72000 && class_exists(Uuid::class)) {
```

Comment:
```suggestion
        if (\PHP_VERSION_ID >= 70200 && class_exists(Uuid::class)) {
```
(not sure why you need it though)